docs/prometheus/README.md

```diff
@@ -1,114 +1,129 @@
-# Using Prometheus with your MongoDB Resource
+# Use Prometheus with your MongoDB Resource
 
-We have added a sample yaml file that you could use to deploy a MongoDB resource
-in your Kubernetes cluster, with a
+ You can use the [mongodb-prometheus-sample.yaml](https://github.com/mongodb/mongodb-kubernetes-operator/blob/master/docs/prometheus/mongodb-prometheus-sample.yaml) file to 
+ deploy a MongoDB resource in your Kubernetes cluster, with a
 [`ServiceMonitor`](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md#related-resources)
-to indicate Prometheus how to consume metrics data from it.
+to indicate to Prometheus how to consume metrics data from 
+it.
 
-This is a simple MongoDB resource with one user, and with the `spec.Prometheus`
-attribute with basic HTTP Auth and no TLS, that will allow you to test
-Prometheus metrics coming from MongoDB.
+The sample specifies a simple MongoDB resource with one user,
+and the `spec.Prometheus` attribute with basic HTTP 
+authentication and no TLS. The sample will allow you to test
+the Prometheus metrics coming from MongoDB.
 
 ## Quick Start
 
-We have tested this setup with version 0.54 of the [Prometheus
+We tested this setup with version 0.54 of the [Prometheus
 Operator](https://github.com/prometheus-operator/prometheus-operator).
 
-### Installing Prometheus Operator
+### Prerequisites
 
-The Prometheus Operator can be installed using Helm. Find the installation
-instructions
-[here](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack#kube-prometheus-stack):
+* Kubernetes 1.16+
+* Helm 3+
 
-This can be done with:
+### Install the Prometheus Operator
+
+You can install the Prometheus Operator using Helm. To learn 
+more, see the [installation instructions](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack#kube-prometheus-stack):
+
+To install the Prometheus Operator using Helm, run the 
+following commands:
 
 ``` shell
 helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
 helm repo update
-helm install prometheus prometheus-community/kube-prometheus-stack --namespace prometheus-system --create-namespace
+helm install prometheus prometheus-community/kube-prometheus-stack --namespace <prometheus-system> --create-namespace
```

Comment:
**suggestion**: this is a lengthy command. consider using a `\` to split it to multiple lines. If you do, indent the second line by two spaces to make it clear that it's a continuation of the same command.